src/OSPSuite.Core/Domain/Services/QuantityValuesUpdater.cs

```diff
@@ -64,6 +64,7 @@ private void updateParameterFromExpressionProfiles(ModelConfiguration modelConfi
 
       private void updateParameterFromIndividualValues(ModelConfiguration modelConfiguration)
       {
+         //Order by distribution to ensure that distributed parameter 
```

Comment:
the comment reminds me of:
<details><summary> 😉 </summary>
<img width="300" src="https://user-images.githubusercontent.com/25061876/230328542-ab7a2e86-5488-4e8b-8fc4-be5e6f58a3e5.png">
</details>